/scripts/simple-fs-read.js:11
  throw new Error('trace');
  ^

Error: trace
    at callback (/scripts/simple-fs-read.js:11:9)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:405:3)
    at ReadFileContext.close (fs.js:318:11)
    at FSReqWrap.readFileAfterRead [as oncomplete] (fs.js:374:15)
    at ReadFileContext.read (fs.js:302:11)
    at FSReqWrap.readFileAfterStat [as oncomplete] (fs.js:358:11)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:334:11)
    at Object.fs.readFile (fs.js:263:11)
    at Object.<anonymous> (/scripts/simple-fs-read.js:10:4)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Function.Module.runMain (module.js:431:10)
    at startup (node.js:141:18)
    at node.js:977:3
